src/OsiXpr/OsiXprSolverInterface.cpp

```diff
@@ -137,8 +149,15 @@ void OsiXprSolverInterface::branchAndBound()
     return;
   }
 #else
+  const char * args = "";
+  bool takeHint;
+  OsiHintStrength strength;
+  getHintParam(OsiDoDualInResolve, takeHint, strength);
+  if (strength != OsiHintIgnore) {
+    args = takeHint ? "d" : "p";
+  }
 
-  XPRS_CHECKED(XPRSmipoptimize, (prob_, ""));
+  XPRS_CHECKED(XPRSmipoptimize, (prob_, args));
```

Comment:
So if one passes 'p' or 'd' here, will that then only solve the LP relaxation, or will Xpress use this argument to decide how to solve the initial LP relaxation in B&B?
If the latter, then taking `OsiDoDualInInitial` may fit better?

Reply:
As I read the docs https://www.fico.com/fico-xpress-optimization/docs/dms2020-02/solver/optimizer/HTML/XPRSmipoptimize.html then 'p' and 'd' is for the initial LP relaxation. One must pass also the 'l' flag to only solve the LP.

`OsiDoDualInInitial` seems like the right choice